src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="body">
+    <div class="body__calendar calendar calendar--month-length--31 calendar--start-week--thu">
```

Comment:
guess you don't need this class
```suggestion
    <div class="calendar calendar--month-length--31 calendar--start-week--thu">
```